<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>161</y>
     <width>256</width>
     <height>179</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>207</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nLabel">
      <property name="text">
       <string>姓名(&amp;N)</string>
      </property>
      <property name="buddy">
       <cstring>nLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="sLabel">
      <property name="text">
       <string>性别(&amp;S)</string>
      </property>
      <property name="buddy">
       <cstring>sComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="sComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="aLabel">
      <property name="text">
       <string>年龄(&amp;A)</string>
      </property>
      <property name="buddy">
       <cstring>aSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="aSpinBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="mLabel">
      <property name="text">
       <string>邮箱地址(&amp;M)</string>
      </property>
      <property name="buddy">
       <cstring>mLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="mLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Button:</string>
   </property>
   <property name="buddy">
    <cstring>pushButton_2</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;CheckBox:</string>
   </property>
   <property name="buddy">
    <cstring>checkBox</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="buddyLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buddyButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>248</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>expand</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sComboBox</tabstop>
  <tabstop>nLineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>buddyButton</tabstop>
  <tabstop>aSpinBox</tabstop>
  <tabstop>mLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
